src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,10 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+18.0.0b30
++++++++
+* Remove interactive AI-powered debugging tool `az aks agent`.
```

Comment:
It is better to add some information about guiding users to download new package to use the related function.